spec/unit/resource/api1000/synergy/volume_attachment_spec.rb

```diff
@@ -0,0 +1,20 @@
+# (C) Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+require 'spec_helper'
+
+RSpec.describe OneviewSDK::API1000::Synergy::VolumeAttachment do
+  include_context 'shared context'
+
+  it 'inherits from OneviewSDK::API1000::C7000::VolumeAttachment' do
+    expect(described_class).to be < OneviewSDK::API1000::C7000::VolumeAttachment
```

Comment:
the lib file for API1000::Synergy is inheriting from API800::Synergy. Make check and make changes in spec file accordingly